plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Any, Dict, List, Optional
+
+from hydra.core.config_store import ConfigStore
+from hydra.types import ObjectConf
+from omegaconf import MISSING
+
+
+@dataclass
+class RayInitConf(Dict[str, Any]):
+    """ parameters for ray.init()"""
+
+    pass
+
+
+@dataclass
+class RayRemoteConf(Dict[str, Any]):
+    """ parameters for @ray.remote() """
+
+    num_cpus: Optional[int] = 1
+    num_gpus: Optional[int] = 0
+
+
+@dataclass
+class RayConf:
+    ray_init_cfg: RayInitConf = field(default_factory=RayInitConf)
+    ray_remote_cfg: RayRemoteConf = field(default_factory=RayRemoteConf)
+
+
+@dataclass
+class RayLocalLauncherConf(ObjectConf):
+    params: RayConf = RayConf()
+
+
+# Ray AWS config
+
+
+class RayAutoScalingMode(Enum):
+    default = "default"
+    aggressive = "aggressive"
+
+
+@dataclass
+class RayDockerConf(Dict[str, Any]):
```

Comment:
Why are you extending Dict?

Reply:
I think i missed this one when cleaning up. will remove.